tests/cpu_bugs/taa.pm

```diff
@@ -0,0 +1,82 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: CPU BUGS on Linux kernel check
+# Maintainer: James Wang <jnwang@suse.com>
+
+package taa;
+use strict;
+use warnings;
+use base "consoletest";
+use base "Mitigation";
+use bootloader_setup;
+use ipmi_backend_utils;
+use power_action_utils 'power_action';
```

Comment:
You don't use power_action here

Reply:
line 67, call reboot_and_wait, but I don't need import 'power_aciton' here, right?

Reply:
right